modules/client/examples/03-tokenVoting-client/18-undelegate-tokens.ts

```diff
@@ -0,0 +1,49 @@
+/* MARKDOWN
+---
+title: Undelegate Tokens
+---
+
+## Undelegates yout token voting power from the delegatee address
+
+This is the same as delegating to your own address.
+*/
+
+import {
+  ContextPlugin,
+  DelegateTokensStep,
+  TokenVotingClient,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+// Instantiate a plugin context from the Aragon OSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(
+  contextPlugin,
+);
+
+
+const steps = tokenVotingClient.methods.undelegateTokens("0x1234567890123456789012345678901234567890");
+
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case DelegateTokensStep.DELEGATING:
```

Comment:
Should it maybe be undelegating? or would you keep it like this?

Reply:
Adding the undelegating would make the code a bit messier since the function only calls the `delegateTokens` function. I'd leave it this way because you aren't doing nothing new. We can add an UndelegateTokenStep that equals the DelegateTokenStep if you want just for syntax sugar reasons But I wouldnt go further

Reply:
Sounds good, let's do an alias that makes things more comprehensive for SDK users